nbs/src/tbats.ipynb

```diff
@@ -894,7 +890,18 @@
     "        log_likelihood = len(y_trans)*np.log(np.nansum(errors**2))\n",
     "\n",
     "    kval = len(optim_params)+x_nought.shape[0]\n",
-    "    aic = log_likelihood+2*kval \n",
+    "    \n",
```

Comment:
I found that some time series "explode" (for example, Y21480 from the M4 dataset). 
This change in how the AIC is computed helps fix some of these cases. 
This was proposed by [De Livera, Hyndman and Snyder](https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=f3de25596ab60ef0e886366826bf58a02b35a44f) on page 20.